troubleshoot/log.js

```diff
@@ -45,18 +45,17 @@ function formatRecordValue(value) {
     }
 
     // Handle boolean
-    if (value.constructor === Boolean) {
+    if (typeof value === 'boolean') {
         return `<span class="boolean">${escapeHTML(value)}</span>`;
     }
 
     // Handle number
-    if (value.constructor === Number) {
+    if (typeof value === 'number' || typeof value === 'bigint') {
         return `<span class="number">${escapeHTML(value)}</span>`;
     }
 
-    // Handle string, converted types (e.g. ArrayBuffer, Blob, ...)
```

Comment:
I would leave this comment as-is since it turned out to make sense (referring to the log format itself)